consensus/core/src/config/params.rs

```diff
@@ -273,3 +300,13 @@ pub const DEVNET_PARAMS: Params = Params {
     max_block_level: 250,
     pruning_proof_m: 1000,
 };
+
+#[cfg(test)]
+mod tests {
+    use crate::config::params::{DIFFICULTY_MAX, DIFFICULTY_MAX_AS_F64};
+
+    #[test]
+    fn test_difficulty_max_consts() {
```

Comment:
Can you also test that `DIFFICULTY_MAX` is `Uint256::from_u64(1).wrapping_shl(255) - 1.into()`?